code/game/machinery/clonepod.dm

```diff
@@ -571,11 +567,8 @@
 			to_chat(user, "<span class='warning'>Access denied.</span>")
 			return
 
-		switch(tgui_alert(user, "Change access restrictions or perform an emergency ejection of [src]?", "Cloning pod", list("Change access", "Emergency ejection")))
-			if("Change access")
-				locked = !locked
-				to_chat(user, "<span class='notice'>Access restriction is now [locked ? "enabled" : "disabled"].</span>")
-			if("Emergency ejection")
+		switch(tgui_alert(user, "Perform an emergency ejection of [src]?", "Cloning pod", list("Yes", "No")))
+			if("Yes")
 				eject_clone(TRUE) // GET OUT
```

Comment:
Within `eject_clone()` I suggest adding some logging as it doesn't appear there is any in the method at the moment. ~line 360 is where the proc is.

Reply:
good idea